src/electron/platform/android/setup/steps/detect-adb.ts

```diff
@@ -7,6 +7,6 @@ export const detectAdb: AndroidSetupStepConfig = deps => ({
     actions: {},
     onEnter: async () => {
         const detected = await deps.hasAdbPath();
-        deps.stepTransition(detected ? 'detect-devices' : 'prompt-locate-adb');
+        deps.stepTransition(detected ? 'prompt-connect-to-device' : 'prompt-locate-adb');
```

Comment:
Our golden path includes finding ADB by environment variables. When that's the case, hasAdbPath will return false but we should attempt to get the devices anyway. We should only go to prompt-locate-adb after detectDevices has failed. Not part of the changes of this PR, and it may also be included in future work, but I wanted to call it out while I was thinking about this path